src/components/ModalBilling/ModalBilling.tsx

```diff
@@ -145,6 +147,8 @@ class ModalBilling extends React.Component<Props, State> {
                   )}
                   onChange={(e) => this.handleChange(e)}
                   value={this.state.stateForm}
+                  maxLength={2}
```

Comment:
Would it be possible to change this to a select, so that we know they type at least the right state? It'll be difficult to propagate a Stripe error back after they hit confirm to instruct them to go and correct their address.